drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -53,6 +57,12 @@ static struct nrf5_802154_data nrf5_data;
 #define FRAME_PENDING_BYTE 1
 #define FRAME_PENDING_BIT (1 << 4)
 
+#if defined(CONFIG_SOC_NRF5340_CPUAPP) || defined(CONFIG_SOC_NRF5340_CPUNET)
+#define EUI64_ADDR (NRF_FICR->INFO.DEVICEID)
```

Comment:
@anangl how do you ensure that these macros are not accessed by non-secure build?